client/me/developer/dev-survey-notice.tsx

```diff
@@ -0,0 +1,74 @@
+import { Gridicon } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import surveyImage from 'calypso/assets/images/illustrations/developer-survey.svg';
+
+type DevSurveyNotice = {
+	onClose: ( remindTimeInSeconds: number ) => void;
+	onOk: () => void;
+	localeSlug: string;
+};
+
+const ONE_DAY_IN_SECONDS = 24 * 60 * 60;
+const ONE_YEAR_IN_SECONDS = 365 * ONE_DAY_IN_SECONDS;
+
+export const DevSurveyNotice = ( { onClose, onOk, localeSlug }: DevSurveyNotice ) => {
```

Comment:
I think we usually suffixes types like this, no?

```suggestion
export const DevSurveyNotice = ( { onClose, onOk, localeSlug }: DevSurveyNoticeProps ) => {
```